api_public.yaml

```diff
@@ -25,7 +25,36 @@ paths:
               version: "0.0.1"
               min_app_version: "0.0.0"
               min_app_version_pagopa: "0.0.0"
+  "password-login":
```

Comment:
```suggestion
  "test-login":
```

Reply:
it looks like api does not lint:

```
ERRORS
  #/paths: Additional properties not allowed: password-login

```

Reply:
Fixed in 48b864c
I've also changed the path including missing base path